win32ss/user/ntuser/cursoricon.c

```diff
@@ -658,6 +658,12 @@ NtUserGetCursorInfo(
     TRACE("Enter NtUserGetCursorInfo\n");
     UserEnterShared();
 
+    if (pci == NULL || (PVOID)pci > MmHighestUserAddress)
```

Comment:
In principle, this should be the job of the ProbeForWrite made down below.
Isn't it possible to move it there instead?

Reply:
This check was added to exit the function if the input parameter is invalid. This check can be removed.